src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,227 @@
+      else:
+        return array[floor((position - start)*count/length)]
+    */
+    array<float> values = [];
+  }
+
+  /*
+  A set of wiggle annotations
+  */
+  record WiggleSet {
+    /** Id of this annotation node. */
+    string id;
+
+    /** Set of additional attributes */
+    Attributes attributes;
+  }
+
+  /**
+  Node in the annotation graph that annotates a contiguous region of a
+  sequence.
+  */
+  record Feature {
+    /**
+    Id of this annotation node.
+    */
+    string id;
+
+    /**
+    Ids of the parents of this annotation node.  
+    */
+    array<string> parentIds;
+
+    /**
+    Identifier for the containing feature set.
+    */
+    string featureSetId;
+
+    /**
+    Genomic location.
+    */
+    /* TODO: path not used yet, as graph code is not working in server.
+       Fields below are copied directly from variants.avdl for now.
+    Path path; */
+
+    /**
+    The reference on which this feature occurs.
+    (e.g. `chr20` or `X`)
+
+    If the API server supports the "classic" mode, this field must not be null.
+    */
+    union { null, string } referenceName = null;
+
+    /**
+    The start position at which this feature occurs (0-based).
+    This corresponds to the first base of the string of reference bases.
+    Genomic positions are non-negative integers less than reference length.
+    Features spanning the join of circular genomes are represented as
+    two features one on each side of the join (position 0).
+
+    If the API server supports the "classic" mode, this field must not be null.
+    */
+    union { null, long } start = null;
+
+    /**
+    The end position (exclusive), resulting in [start, end) closed-open interval.
+    This is typically calculated by `start + referenceBases.length`.
+
+    If the API server supports the "classic" mode, this field must not be null.
+    */
+    union { null, long } end = null;
+
+    /**
+    Feature that is annotated by this region.  Normally, this will be a term in
+    the Sequence Ontology.
+    */
+    OntologyTerm featureType;
+
+    /**
+    Name/value attributes of the annotation.  Attribute names follow the GFF3
+    naming convention of reserved names starting with an upper cases
+    character, and user-define names start with lower-case.  Most GFF3
+    pre-defined attributes apply, the exceptions are ID and Parent, which are
+    defined as fields. Additional, the following attributes are added:
+    * Score - the GFF3 score column
+    * Phase - the GFF3 phase column for CDS features.
+    */
+    Attributes attributes;
+  }
+
+  /*
+  A set of sequence features annotations
+  */
+  record FeatureSet {
+    /** The ID of this annotation set. */
+    string id;
+
+    /** The ID of the dataset this annotation set belongs to. */
+   union { null, string } datasetId = null;
+
+    /**
+    The ID of the reference set which defines the coordinate-space for this
+    set of annotations.
+    */
+    union { null, string } referenceSetId;
```

Comment:
I know I'm breaking ranks here, but I increasingly feel that in a truly distributed system, this shouldn't be an ID but rather a universally curated (enumerated) referenceSetName (such as `GRCh38`)